[(<class 'UnboundLocalError'>, UnboundLocalError("local variable 'j' referenced before assignment"), <traceback object at 0x7fd7080d3dc0>), (<class 'TypeError'>, TypeError("run_experiment_one() got an unexpected keyword argument 'data_to_test_one'"), <traceback object at 0x7fd7080d3cc0>), (<class 'UnboundLocalError'>, UnboundLocalError("local variable 'j' referenced before assignment"), <traceback object at 0x7fd6f039ba00>)]